.github/workflows/check-duplicate-images.yml

```
@@ -40,9 +40,16 @@ jobs:
         uses: peter-evans/create-pull-request@v6
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
+          committer: SSW-Rules-Bot <rules-bot@example.com>
+          author: SSW-Rules-Bot <rules-bot@example.com>
           commit-message: Auto Rename Duplicate Images
           title: Auto Rename Duplicate Images
-          body: Generated Automatically via GitHub Actions
+          body: |
+            Generated Automatically via GitHub Actions.
+
+            This PR was created because automated checks found duplicate image names in Rules.Content and tried to rename them.
+
+            The aim of doing this is to prevent incorrect images from being displayed on the Rule page, thus minimizing potential future maintenance issues. See detail at https://www.github.com/SSWConsulting/SSW.Rules/issues/846
```

I think this is a bit clearer
```suggestion
            The goal is to prevent incorrect images from being displayed on rules, see detail at https://www.github.com/SSWConsulting/SSW.Rules/issues/846
```

what would be even better is a wiki page